content/kubernetes/re-clusters/delete_custom_resources.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: Delete custom resources
+linkTitle: Delete custom resources
+description: This article explains how to delete Redis Enterprise clusters and Redis Enterprise databases from your Kubernetes environment.
+weight: 90
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/delete_custom_resources.md,
+    /kubernetes/re-clusters/delete_custom_resources/,
+]
+---
+
+## Delete a database
+
+To delete a database managed by the Redis Enterprise Kubernetes operator, run `kubectl delete redb <your-db-name>` from your K8s cluster.
+
+When you delete a database, your data and the REDB custom resource file are also deleted.
+
+### Troubleshoot REDB deletion
+
+After a deletion request, the operator attaches a finalizer to the Redis Enterprise database (REDB) object. This makes sure the database is deleted before the REDB custom resource is removed from the K8s cluster.
+
+If the finalizer isn't removed automatically by the operator, you won't be able to delete your REDB resource.
+
+If this happens, you can remove the finalizer manually with the following command:
+
+```sh
+kubectl patch redb <your-db-name> --type=json -p \
+    '[{"op":"remove","path":"/metadata/finalizers","value":"finalizer.redisenterprisedatabases.app.redislabs.com"}]'
+```
+
+## Delete a Redis Enterprise cluster (REC)
+
+To delete a Redis Enterprise cluster managed by the operator:
+
+1. Delete all the databases in your cluster
+
+1. Delete all the PVC's you created for the cluster.
```

Comment:
I believe the PVCs deletion should take place only after the REC is successfully deleted, as it's still mounted to the REC nodes.

Also: I think that some PVCs are auto-created for the user in the case of RoF (@heinrich-redislabs or @yuvallevy2 can you confirm this?) and are not auto-deleted, so it could be useful to instruct the user how to locate them.

Reply:
@laurentdroin maybe you can help us figure out if the PVCs should be deleted before or after the REC?

Reply:
Looked some more into this and also validated with @heinrich-redislabs. Here's the deal:
1. The PVCs _may_ be deleted before the REC is deleted. In this case, the PVCs will be "marked for deletion" but the  deletion will only take place after the consuming pods (the REC pods) are deleted ([source](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection)).
2. Deleting the PVCs after the REC is deleted, will result in immediate deletion.
3. To keep the docs simple and straight forward, we should choose only one of these options. I think the 2nd option (after) is preferable since it spares the users from asking themselves "what happens to my PVCs if I delete them while in use".
4. Regardless, I think the instructions should let the user know he _may_ delete the PVCs, but may also choose to retain them for the purpose of future recovery/restoration (I'm not sure what's the right user-facing term in use).
5. Here's the command to delete the PVCs associated with a specific REC, if you like to include it:
```
kubectl delete pvc -l redis.io/cluster=<your-rec-name>
```